x-pack/plugins/security_solution/public/common/components/event_details/get_alert_summary_rows.tsx

```diff
@@ -215,10 +215,15 @@ function getFieldsByRuleType(ruleType?: string): EventSummaryField[] {
   }
 }
 
+/**
+  This function is exported because it is used in the Exception Component to
+  populate the conditions with the Highlighted Fields. Additionally, the new 
+  Alert Summary Flyout also requires access to these fields.
+ */
```

Comment:
@PhilippeOberti fyi 😊

Reply:
Wonder if it's worth moving out to one of our packages?

Reply:
@PhilippeOberti and I discussed the possibility of relocating this method and its helpers to a more appropriate location and renaming it to provide better contextual information. However, Philippe informed me that they are already in the process of refactoring the `Alert Summary` component, which means these files will undergo changes soon. 

To avoid working on code that may undergo further modifications, we decided to proceed with minimal changes for now and add a comment explaining the reason for exporting the function. I can also update the comment to provide further clarity on the situation mentioned above :)

Reply:
Yeah I talked about it, and like @WafaaNasr  said it seemed to us to the easiest path for now was the minimum amount of code changes.
I'm not super familiar with this whole part of the code, so take my comment with a grain of salt, but I feel like moving things to a package should require a plan, and the scope of this PR could suffer quite a bit from it... I'm definitely not against it though!

Reply:
The whole concept of "highlighted fields" seems to be expanding. There's also a PR out from a community member that would allow a user to specify what fields they want to be considered as "highlighed fields" so it may be worth syncing up with you all (threat hunting) on any future changes there. If you could keep us in the loop that'd be appreciated!